tests/integration_test.rs

```diff
@@ -310,6 +316,36 @@ async fn test_reproducible_shasum(options: Options) {
     let pack_result = pixi_pack::pack(pack_options.clone()).await;
     assert!(pack_result.is_ok(), "{:?}", pack_result);
 
+    // Second pack.
+    pack_options.output_file = output_file2.clone();
+    let pack_result = pixi_pack::pack(pack_options.clone()).await;
+    assert!(pack_result.is_ok(), "{:?}", pack_result);
+
+    assert_eq!(
```

Comment:
i found a nicer way to test reproducibility using insta, see #75, so this test needs to be adjusted when merging main 😓 sorry for the extra effort